### PR TITLE
ACM plugin custom extension for row action

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by KevinFCormier** [comment]
@GowthamShanmugam Please run `npm run prettier:fix` in the `frontend` directory. Thanks!

**Comment by GowthamShanmugam** [comment]
> @GowthamShanmugam Please run `npm run prettier:fix` in the `frontend` directory. Thanks!

ack

File: frontend/src/routes/Applications/Overview.tsx [hunk 1]
```diff
@@ -871,6 +875,29 @@ export default function ApplicationsOverview() {
                 })
             }
 
+            if (acmExtensions) {
+                if (acmExtensions?.applicationAction?.length) {
```
Comment (SanjalKatiyar) [inline frontend/src/routes/Applications/Overview.tsx:879]:
```suggestion
                if (acmExtensions?.applicationAction?.length) {
```

File: frontend/src/routes/Applications/Overview.tsx [hunk 2]
```diff
@@ -871,6 +875,29 @@ export default function ApplicationsOverview() {
                 })
             }
 
+            if (acmExtensions) {
```
Comment (SanjalKatiyar) [inline frontend/src/routes/Applications/Overview.tsx:878]:
```suggestion
```

**Review by fxiang1 — APPROVED** [review]
LGTM!

**Comment by fxiang1** [comment]
Looks like there's some lint issues from the unit test:
```
 [frontend] [prettier] npm run prettier exited with code 0
[frontend] [tsc] src/routes/Applications/Overview.tsx(8,65): error TS6133: 'ElementType' is declared but its value is never read.
[frontend] [tsc] src/routes/Applications/Overview.tsx(8,78): error TS6133: 'ComponentType' is declared but its value is never read.
[frontend] [tsc] npm ERR! code ELIFECYCLE
[frontend] [tsc] npm ERR! errno 2
[frontend] [tsc] npm ERR! frontend@0.0.1 tsc: `tsc --noEmit` 
```

I think fixing those should pass the tests.

**Comment by GowthamShanmugam** [comment]
@fxiang1 all check has passed

**Review by fxiang1 — APPROVED** [review]
LGTM!

**Comment by fxiang1** [comment]
@GowthamShanmugam Approved. The PR should merge automatically when you remove the hold label.

**Comment by GowthamShanmugam** [comment]
@fxiang1 Thanks for the review

**Comment by fxiang1** [comment]
FYI @thuyn-581 There's no issue attached with this UI change so I'm not sure if QE was notified and if the test automation can handle the change.

**Pull request merged**